_data/jobs.yml

```
@@ -1,29 +1,29 @@
-- title: Ruby Developer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/202168?language=en
-  description: With your skills in Ruby, Rails, JavaScript, and PostgreSQL you are responsible for the development of our crucial in-house tools.
-  location: Bilbao
+- title: Frontend Developer (m/f/d)
+  link: https://epages-gmbh.jobs.personio.de/job/224109
+  description: With your sense for modern UI design and aesthetics you create the visual appearance of our products and drive the visual identity.
+  location: Hamburg
 
 - title: Senior Java Developer (m/f/d)
   link: https://epages-gmbh.jobs.personio.de/job/202115
   description: Bring your knowledge of Java and Kotlin to work on a microservice-based architecture for our new ecommerce platform.
   location: Hamburg
 
 - title: JavaScript Engineer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/201909?language=en
+  link: https://epages-gmbh.jobs.personio.de/job/201909
   description: Build a responsive, modular, and highly user interactive web application with us that is easy to scale and to extend.
   location: Hamburg
 
-- title: Software Developer - 3rd Party Integration, REST API (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/202173?language=en
-  description: Develop connectors that integrate third-party systems, such as payment gateways or product portals, into our ecommerce platform.
-  location: Barcelona
+- title: Technical Writer - API Docs & Social Media (m/f/d)
+  link: https://epages-gmbh.jobs.personio.de/job/522686
+  description: Empower developers to create awesome apps by taking care of our API documentation and get creative by creating content for our social media channels.
+  location: Hamburg
 
 - title: Technical Product Owner (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/226186?language=en
-  description: With your knowledge of the tech stacks of website and commerce platforms you are responsible for several Scrum teams.
+  link: https://epages-gmbh.jobs.personio.de/job/226186
+  description: Be responsible for the further development of our software and support several Scrum teams with your knowledge.
   location: Hamburg
 
-- title: Full Stack Developer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/384247?language=
-  description: Be part of a cross-functional team and use your Java and/or TypeScript skills to further develop our new ecommerce platform.
+- title: User Experience Designer (m/f/d)
+  link: https://epages-gmbh.jobs.personio.de/job/203174
+  description: Keep an overview of the UX of all products and develop user-friendly interaction concepts by working closely with our product management team.
```

You are already using "by" a lot for the Tech Writer position. Thus, I'd go with while here :)

```suggestion
  description: Keep an overview of our products' UX and develop user-friendly interaction concepts while working closely with our product management team.
```